marker/providers/pdf.py

```diff
@@ -172,6 +179,47 @@ def check_line_spans(self, page_lines: List[ProviderOutput]) -> bool:
             return False
         return True
 
+    def check_page(self, page_id: int) -> bool:
+        page = self.doc.get_page(page_id)
+        page_bbox = PolygonBox.from_bbox(page.get_bbox())
+        page_objs = list(page.get_objects(filter=[pdfium_c.FPDF_PAGEOBJ_TEXT, pdfium_c.FPDF_PAGEOBJ_IMAGE]))
+
+        # if we do not see any text objects in the pdf, we can skip this page
+        if not any([obj.type == pdfium_c.FPDF_PAGEOBJ_TEXT for obj in page_objs]):
```

Comment:
Oh this might be redundant, this just means that we don't have any characters in the PDF that can be extracted. This would end up going to the OCR either way because the provider would have no lines, e.g `A17FlightPlan.pdf`, this condition is jut doing it sooner